tests/presentations/indexable-post-type-presentation/og-description-test.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+namespace Yoast\WP\Free\Tests\Presentations\Indexable_Post_Type_Presentation;
+
+use Yoast\WP\Free\Tests\TestCase;
+
+/**
+ * Class Open_Graph_Description_Test
+ *
+ * @coversDefaultClass \Yoast\WP\Free\Presentations\Indexable_Post_Type_Presentation
+ *
+ * @group presentations
+ * @group open-graph
+ */
+class Open_Graph_Description_Test extends TestCase {
+	use Presentation_Instance_Builder;
+
+	/**
+	 * Does the setup for testing.
+	 */
+	public function setUp() {
+		$this->setInstance();
+		$this->indexable->object_id = 1;
+
+		parent::setUp();
```

Comment:
I must confess I use the same pattern at some place, but I prefer calling the parent first before doing custom things.

Reply:
Agreed. Copy-paste result 😉